Code/Tools/TestImpactFramework/Runtime/Code/Include/TestImpactFramework/TestImpactClientSequenceReport.h

```diff
@@ -382,6 +370,16 @@ namespace TestImpact
             {
             }
 
+            DraftingSequenceReportBase(
+                SequenceReportBase<Type, PolicyStateType>&& report,
+                const AZStd::vector<AZStd::string>& draftedTestRuns,
```

Comment:
minor: This vector could be passed by value and then `AZStd::moved` into the member as it's a sink parameter

Reply:
fixed 👍